ClosedXML/Excel/PivotTables/IXLPivotTable.cs

```diff
@@ -126,7 +129,10 @@ public interface IXLPivotTable
         String RowHeaderCaption { get; set; }
 
         IXLCell TargetCell { get; set; }
+
         IXLRange SourceRange { get; set; }
+        IXLTable SourceTable { get; set; }
```

Comment:
Apply auto formatting with CodeMaid once more?

Reply:
I did. And redid it now. No change. What looks wrong?

Reply:
All the properties have empty lines between them but these three stand tight so I thougth this line was added after the auto formatting was performed.

Reply:
Oh, I thought you meant the indentation or something was wrong. It seems CodeMaid doesn't add new lines between elements of interfaces. I left these three grouped together because they're kind of related.

Reply:
Ah, OK then

Reply:
![image](https://user-images.githubusercontent.com/145854/39705927-19258b3a-5210-11e8-8934-fbd1d44c1ba8.png)

This is why CodeMaid doesn't fix that. Going to leave it as is for now.